nanovdb/nanovdb/CMakeLists.txt

```diff
@@ -322,16 +322,16 @@ if(TARGET Threads::Threads)
   target_link_libraries(nanovdb INTERFACE Threads::Threads)
 endif()
 
-set(NANOVDB_INSTALL_INCLUDE_DIR ${CMAKE_INSTALL_INCLUDEDIR}/nanovdb)
-set(NANOVDB_INSTALL_CUDA_DIR ${NANOVDB_INSTALL_INCLUDE_DIR}/cuda)
-set(NANOVDB_INSTALL_IO_DIR ${NANOVDB_INSTALL_INCLUDE_DIR}/io)
-set(NANOVDB_INSTALL_MATH_DIR ${NANOVDB_INSTALL_INCLUDE_DIR}/math)
-set(NANOVDB_INSTALL_TOOLS_DIR ${NANOVDB_INSTALL_INCLUDE_DIR}/tools)
+set(NANOVDB_INSTALL_ROOT_DIR ${NANOVDB_INSTALL_INCLUDEDIR}/nanovdb)
+set(NANOVDB_INSTALL_CUDA_DIR ${NANOVDB_INSTALL_ROOT_DIR}/cuda)
```

Comment:
There are a few of these variables that are swapping INCLUDE paths for ROOT paths?

Reply:
Not a typo I think.

Prior to this change, `NANOVDB_INSTALL_INCLUDE_DIR` (note the underscore between include and dir) was already defined. I've renamed this variable to `NANOVDB_INSTALL_INCLUDE_DIR` to `NANOVDB_INSTALL_ROOT_DIR` in order to avoid confusion with the newly introduced `NANOVDB_INSTALL_INCLUDEDIR` (that defaults to `CMAKE_INSTALL_INCLUDEDIR`).

Reply:
Ah, that makes sense, thanks for explaining.